javacutil/src/main/java/org/checkerframework/javacutil/Resolver.java

```diff
@@ -79,7 +82,13 @@ public class Resolver {
               boolean.class);
       FIND_METHOD.setAccessible(true);
 
-      FIND_VAR = Resolve.class.getDeclaredMethod("findVar", Env.class, Name.class);
+      if (atLeastJava23) {
```

Comment:
Does this fix https://github.com/typetools/checker-framework/issues/6704?

Reply:
Yes.  I linked it.